src/Time/TimeSteppers/AdamsBashforthN.hpp

```diff
@@ -45,9 +45,52 @@ class History;
 
 namespace TimeSteppers {
 
-/// \ingroup TimeSteppersGroup
-///
-/// An Nth Adams-Bashforth time stepper.
+/*!
+ * \ingroup TimeSteppersGroup
+ *
+ * An Nth order Adams-Bashforth time stepper.
+ *
+ * The stable step size factors for different orders are given by:
+ *
+ * <table class="doxtable">
+ *  <tr>
```

Comment:
Would a markdown-style table work here, just to make the source code shorter?  I.e.,
```
Order | CFL Factor
------------------
1     | 1
2     | 1/2
```

Reply:
I tried this and was unsuccessful in convincing Doxygen to render anything that looked remotely like columns and rows...

Reply:
Figured it might be doxygen screwiness.